05-mvc-with-rails/rails-views-and-controllers/readme.md

```diff
@@ -192,17 +300,33 @@ def create
     redirect_to "/posts/new"
   end
 end
-
-def update
-  @artist = Artist.find(params[:id])
-  @artist.update(artist_params)
-  redirect_to "/artists/#{@artist.id}"
-end
 ```
 
 Great, now we're protected!
 
-## You do - (30/125)
-- Incorporate strong params for the song model now.
+## You-Do: (15/120)
+- Define `edit`, 'update' and `destroy` controller actions.
+- Create an `edit` view file.
+- When you visit the `edit` page, the relevant form should be displayed
+- Upon submitting the form, artist info updates accordingly.
+- Create a way to destroy an artist
+
+## Break (10/130)
+
+## You-Do: Songs (15/145)
+- Define 7 RESTful routes and map to appropriate songs controller actions.
+- Create a songs controller
+- Define 7 CRUD actions
+- Create appropriate view files
+
+## Closing (5/150)
+
+Who can remind us of the importance of a controller?
+How does the view access data?
+What is the difference between render and redirect?
+How does strong params protect us from malicious input?
 
```

Comment:
Noticing there aren't any industry examples in here... maybe there's an opportunity to talk about MVC architecture as an artifact of the beginnning of GUI programming - https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller#History